ocp_install_env.sh

```diff
@@ -13,7 +13,7 @@ export EXTERNAL_SUBNET="192.168.111.0/24"
 # The release we default to here is pinned and known to work with our current
 # version of kni-installer.
 #
-export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="registry.svc.ci.openshift.org/ocp/release:4.0.0-0.ci-2019-04-17-133604"
+export OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE="registry.svc.ci.openshift.org/ocp/release:4.1.0-rc.3"
```

Comment:
Looks like from the conversation that we agree to use the URL listed on e.g. https://openshift-release.svc.ci.openshift.org/releasestream/4-stable/release/4.1.0-rc.1 - i.e. `quay.io/openshift-release-dev/ocp-release:4.1.0-rc.1`